social_core/tests/backends/test_orcid.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from .oauth import OAuth2Test
+
+from social_core.backends.orcid import ORCIDOAuth2
+
+class ORCIDOAuth2Test(OAuth2Test):
+    backend_path = 'social_core.backends.orcid.ORCIDOAuth2'
+    user_data_url = 'https://login.orbi.kr/oauth/user/get'
```

Comment:
Start with this, it will mock that URL. 

```suggestion
    user_data_url = ORCIDOAuth2.USER_ID_URL
```

You will probably have to modify the other mock params below (I have not checked that).

Reply:
@nijel We have modified these. Will push up amendments.